src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -2461,21 +2461,21 @@ protected override void HandleDataBreakpointInfoRequestAsync(IRequestResponder<D
                         response.AccessTypes = new List<DataBreakpointAccessType>() { DataBreakpointAccessType.Write };
                     }
                 }
-                else if (response.Description == null)
+                else
                 {
-                    response.Description = string.Format(CultureInfo.CurrentCulture, AD7Resources.Error_DataBreakpointInfoFail, AD7Resources.Error_ChildPropertyNotFound);
+                    throw new ProtocolException(string.Format(CultureInfo.CurrentCulture, AD7Resources.Error_DataBreakpointInfoFail, AD7Resources.Error_ChildPropertyNotFound));
```

Comment:
following up on lunch convo: should we throw an exception here?

Reply:
Yes. DataBreakpointInfo does not have `verified`, thats on the SetDataBreakpoint call.